contracts/BondFactory.sol

```diff
@@ -167,6 +176,7 @@ contract BondFactory is IBondFactory, AccessControl {
         );
         // Greater than instead of != for the case where the collateralToken
         // is sent to the clone address before creation.
+        // reverts if the collateralToken takes a fee
```

Comment:
I'm not sure on this. It feels like it could be better articulated. something like. `Check that the amount of collateral in the contract is at least the expected amount deposited. A token could take a fee upon transfer or collateral could be sent to the clone address before creation.`

Is that true? I know there's deterministic address prediction, but I didn't know it could work like this through a clone instantiation.

Reply:
> collateral could be sent to the clone address before creation.

I think i'm going to just remove this. It's possible, but so statistically unlikely I think it's on the verge of impossible. Plus if it does happen the transaction just reverts which isn't a big deal here.

Reply:
I removed in [1422820](https://github.com/porter-finance/v1-core/pull/247/commits/142282054caa9dc0fcd24001dfe032c096ee02a9)